src/components/logger/row.js

```diff
@@ -68,12 +69,11 @@ export default class Row extends React.Component {
 
```

Comment:
you don't use Timestamp const

Reply:
and you don't use import { pick } from 'lodash';

Reply:
done